Signal/test/phone/PhoneNumberTest.m

```diff
@@ -1,155 +0,0 @@
-//
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
-//
-
-#import "OWSContactsManager.h"
-#import "PhoneNumber.h"
-#import "PhoneNumberUtil.h"
-#import "TestUtil.h"
-#import "TextSecureKitEnv.h"
-#import <XCTest/XCTest.h>
-
-@interface PhoneNumberTest : XCTestCase
-
-@end
-
-@implementation PhoneNumberTest
-
--(void) testE164 {
-    test([[[PhoneNumber tryParsePhoneNumberFromText:@"+1 (902) 555-5555" fromRegion:@"US"] toE164] isEqualToString:@"+19025555555"]);
-    test([[[PhoneNumber tryParsePhoneNumberFromText:@"1 (902) 555-5555" fromRegion:@"US"] toE164] isEqualToString:@"+19025555555"]);
-    test([[[PhoneNumber tryParsePhoneNumberFromText:@"1-902-555-5555" fromRegion:@"US"] toE164] isEqualToString:@"+19025555555"]);
-    test([[[PhoneNumber tryParsePhoneNumberFromText:@"1-902-５５５-5555" fromRegion:@"US"] toE164] isEqualToString:@"+19025555555"]);
-}
-
--(void) testTranslateCursorPosition {
-    testThrows([PhoneNumberUtil translateCursorPosition:0 from:nil to:@"" stickingRightward:true]);
```

Comment:
Given the chance, I'd love to completely rework translateCursorPosition.  It's quite complicated and unnecessary.  We can easily compute the correct cursor position by filtering substrings of the formatted output.